core/src/main/java/hudson/console/HyperlinkNote.java

```diff
@@ -75,8 +76,19 @@ protected String extraAttributes() {
     }
 
     public static String encodeTo(String url, String text) {
+        return encodeTo(url, text, HyperlinkNote::new);
+    }
+
+    static String encodeTo(String url, String text, BiFunction<String, Integer, ConsoleNote> constructor) {
```

Comment:
restrict so that Core components/detached plugins do not use it?

Reply:
Done